src/transformState.js

```diff
@@ -62,7 +62,30 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const input of transforms) {
+    const operation = input.operation;
+    const props = input.properties;
```

Comment:
You can do destructuring of input

Reply:
No, I can't. With **const {operation, props} = input;** the array becomes not iterable,
I don't know any other way to do this.